src/Repository/CaseEntityRepository.php

```diff
@@ -135,4 +137,77 @@ public function findCountOfCasesWithNewHearingPostBy(array $criteria): int
         // TODO: Update beneath when hearing stuff has been implemented
         return -1;
     }
+
+    public function findCasesForBoardMember(BoardMember $boardMember)
+    {
+        $qb = $this
+            ->createQueryBuilder('c')
+            ->leftJoin('c.agendaCaseItems', 'aci')
+            ->leftJoin('aci.agenda', 'a')
+            ->andWhere(':boardMember MEMBER OF a.boardmembers')
+            ->setParameter('boardMember', $boardMember->getId()->toBinary())
+            ->orWhere('c.currentPlace = :case_finished_status')
+            ->setParameter('case_finished_status', 'Afgørelse')
+        ;
+
+        return $qb->getQuery()->getResult();
+    }
```

Comment:
Is this function used? If so, it should call `updateQueryBuilderWithBoardFinishStatuses` to make sure that only the right cases are found.

Reply:
Removed in 20abc6d0e1bf0336ec75f36b6613c018bc759b2c as it is never used.